### PR TITLE
Translated to Nepali docs

### PR DESCRIPTION
**Pull request by Aju100**

### CODE REVIEW

**Comment by anuraghazra** [comment]
Hi @Aju100 thanks for the contribution.

Can you complete the translation since i can see some gaps on the translation.

**Comment by Aju100** [comment]
Hi @anuraghazra there, sure.

**Comment by Aju100** [comment]
@anuraghazra  Let me know which section should be translated as some language or sentence can't be changed in Nepali like English terms.

File: docs/readme_np.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+
+```md
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&hide=contribs,prs)
+```
+
+### जोड्नु होस् निजी टोटल  योगदान
+
+You can add the count of all your private contributions to the total commits count by using the query parameter `?count_private=true`.
+
+_Note: If you are deploying this project yourself, the private contributions will be counted by default otherwise you need to chose to share your private contribution counts._
```
Comment (anuraghazra) [inline docs/readme_np.md:94]: translation needed

File: docs/readme_np.md [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+```md
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true)
+```
+
+### विषयवस्तुहरू
+
+With inbuilt themes, you can customize the look of the card without doing any [manual customization](#customization).
+
+Use `?theme=THEME_NAME` parameter like so :-
+
```
Comment (anuraghazra) [inline docs/readme_np.md:115]: translation needed

File: docs/readme_np.md [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+- `icon_color` - Icons color if available _(hex color)_
+- `bg_color` - Card's background color _(hex color)_ **or** a gradient in the form of _angle,start,end_
+- `hide_border` - Hides the card's border _(boolean)_
+- `theme` - name of the theme, choose from [all available themes](./themes/README.md)
+- `cache_seconds` - set the cache header manually _(min: 1800, max: 86400)_
+- `locale` - set the language in the card _(e.g. cn, de, es, etc.)_
+
+##### Gradient in bg_color
+
+You can provide multiple comma-separated values in bg_color option to render a gradient, the format of the gradient is :-
```
Comment (anuraghazra) [inline docs/readme_np.md:146]: translation needed

File: docs/readme_np.md [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+GitHub extra pins allow you to pin more than 6 repositories in your profile using a GitHub readme profile.
+
+Yay! You are no longer limited to 6 pinned repositories.
+
+### रयोग 
+
+Copy-paste this code into your readme and change the links.
+
+Endpoint: `api/pin?username=anuraghazra&repo=github-readme-stats`
+
```
Comment (anuraghazra) [inline docs/readme_np.md:205]: translation needed

File: docs/readme_np.md [hunk 5]
```diff
@@ -0,0 +1,404 @@
...
+The top languages card shows a GitHub user's top languages which have used the most.
+
+_NOTE: Top Languages does not indicate my skill level or anything like that, it's a GitHub metric of which languages have the most code on GitHub. It's a new feature of github-readme-stats._
+
+### प्रयोग 
+
+Copy-paste this code into your readme and change the links.
+
+Endpoint: `api/top-langs?username=anuraghazra`
```
Comment (anuraghazra) [inline docs/readme_np.md:228]: translation needed

File: docs/readme_np.md (hunk 1 above)
Comment (Aju100) [reply docs/readme_np.md:94]: sure

**Comment by Aju100** [comment]
@anuraghazra  thank u for it. I will fix it.

File: docs/readme_np.md (hunk 5 above)
Comment (Aju100) [reply docs/readme_np.md:228]: @anuraghazra  sure

**Comment by Aju100** [comment]
@anuraghazra  I told someone of them is mostly English and hard to explain in the Nepali version

**Comment by Aju100** [comment]
@anuraghazra  any updates ???

**Comment by anuraghazra** [comment]
Merging this then we can improve progressively later on. Thanks @Aju100

**Pull request merged**